Added npm and event collection (#183)
scarf-sh/docs: docs/packages.md

![New file package route](assets/pics/qs-file-packages/file-package-new-route.png)

## npm Packages

NPM Packages on Scarf are a convenient way to collect usage and event telemetry from your npm packages by adding a dependncy on `[scarf-js](https://www.npmjs.com/package/@scarf/scarf)`.

1. Log in to your Scarf account.

2. Click the plus icon in the navigation, then select New Package.
![Create a new package](assets/pics/qs-file-packages/create-new-package.png)

> 3. In the first drop-down click on the package type you would like to create. For this section you will click `npm`.

Reviewer: ```suggestion
3. In the first drop-down, select `npm` as the package type. 
```

![Create a package](assets/pics/qs-file-packages/create-file.png)

4. Select the package owner from the dropdown list of Organizations you belong to.
![Select package owner](assets/pics/qs-file-packages/file-package-select-owner.png)

5. Give your npm package a name.

> This name should match the name of your package in the npm repository.

Reviewer: ```suggestion
This name should match the name of your package on npm, or the same as the `name` in your `package.json`.
```
